lib/hexpm/accounts/session.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Hexpm.Accounts.Session do
```

Comment:
Theoretically you always go over the context: `Hexpm.Accounts.sessions_by_key`. It is a minor detail and one that is not respected in Hexpm right now anyway. /cc @wojtekmach